.travis.yml

```diff
@@ -40,3 +40,9 @@ notifications:
 branches:
   except:
     - /^*-v[0-9]/
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
+     docker login -u "$DOCKER_USER" -p "$DOCKER_PASS";
```

Comment:
We need to move this  block to `script:`. If any command fails in `after_success:`, travis build will not be considered as failed build.

Reply:
Will update it @mynktl

Reply:
AFAIK, it will be considered as failed build if it fails on master branch because we are not running it for pull request.

Can you please explain how `If any command fails in after_success:, travis build will not be considered as failed build.` this works?

Reply:
Will update it @mynktl